haskell-customize.el

```diff
@@ -324,6 +324,17 @@ The default is `haskell-interactive-prompt' with the last > replaced with |."
   :type 'string
   :group 'haskell-interactive)
 
+(defcustom haskell-interactive-use-interactive-prompt t
```

Comment:
@gracjan  @Pitometsu 
I've been trying to get another version that uses `haskell-interactive-prompt` only (checks if it's `nil`), but it's uglier. Also, `haskell-interactive-mode-multi-line` doesn't work without knowing the length of the last part of the prompt. (That is, mult-line with `set +m` and `:{ :}` works, but not with `C-j`).

If you don't mind both of these things, then I can replace this version.

Oh, I also don't see `haskell-doc`/`eldoc` working with it as well.